shopfloor_mobile_base/static/wms/src/services/config_registry.js

```diff
@@ -131,12 +148,26 @@ export class ConfigRegistry {
      * @param {*} v
      */
     set_value(k, v) {
-        // TODO: merge values for objects
-        // const config = this.get(k);
-        // let val = config._safe_value(v);
-        // const new_data = _.merge({}, this.root[this.data_key], {[this.key]: v});
-        this._set_root_val(k, v);
-        this._set_storage_val(k, v);
+        const config = this.get(k);
+        this._set_root_val(config, v);
+        this._set_storage_val(config, v);
+    }
+
+    _switch_storage_driver(driver) {
+        // The app uses sessionStorage by default (currently set on app creation).
+        // If any piece of data needs to be handled by localStorage,
+        // we switch the vue2storage driver option to "local", we store / retrieve the value,
+        // and then we revert to "session" for further use.
+        // See example in documentation: https://github.com/yarkovaleksei/vue2-storage/blob/master/docs/en/started.md.
+        const permitted_types = ["local", "session", "memory"];
```

Comment:
please use https://github.com/yarkovaleksei/vue2-storage/blob/d48b1321bb97359129820e41d2ec3856edd05758/src/storage.ts#L10

Reply:
Cannot be used, as the current latest version of the library doesn't export it:
[See: ](https://unpkg.com/vue2-storage@6.1.3/dist/vue2-storage.js)

Reply:
ok, move this check to config key init